common/src/java/org/apache/hadoop/hive/conf/HiveConf.java

```diff
@@ -3254,6 +3254,8 @@ private static void populateLlapDaemonVarsSet(Set<String> llapDaemonVarsSetLocal
 
     TRANSACTIONAL_CONCATENATE_NOBLOCK("hive.transactional.concatenate.noblock", false,
         "Will cause 'alter table T concatenate' to be non-blocking"),
+    ENABLE_CONCATENATE_FOR_EXTERNAL_TABLES("hive.concatenate.enable-external-tables", false,
+        "Enable concatenate for external tables"),
```

Comment:
What is the expected behaviour of this one? Add some description

Reply:
Fixed.